dm/syncer/syncer.go

```diff
@@ -1560,6 +1546,14 @@ func (s *Syncer) updateTSOffset(ctx context.Context) error {
 
 // Run starts running for sync, we should guarantee it can rerun when paused.
 func (s *Syncer) Run(ctx context.Context) (err error) {
+	if !s.schemaLoaded.Load() {
+		err = s.checkpoint.LoadIntoSchemaTracker(ctx, s.schemaTracker)
```

Comment:
maybe put this load func into sync.Init

Reply:
init may timeout and cause task fail, that's why we add this [pr](https://github.com/pingcap/dm/pull/1271)